concepts/stable-seekers/faq-stable.md

```diff
@@ -4,29 +4,35 @@ description: Common questions about getting and exchanging the Ppotocol's stable
 
 # FAQ - Stable Seekers and Holders
 
+## What is agEUR backed by?
+
+First, agEUR is backed by the collateral that was used to mint it. Then, if this collateral is volatile, HA can open positions on the collateral stored in the protocol to protect it from changes in collateral/agEUR exchange rate. In case this is not enough, SLPs come to over-collateralize the protocol.
+
 ## Could I use the protocol as a decentralized exchange?
 
 Angle can be used as a decentralized exchange where there is no slippage for swapping.
 
-If Angle's agEUR is backed by wETH and wBTC, a user wishing to swap wETH against wBTC with no slippage can choose to come to Angle, swap wETH against agEUR, and then use these agEUR to redeem wBTC.
+For example, if Angle's agEUR is backed by wETH and wBTC, a user wishing to swap wETH against wBTC with no slippage can choose to come to Angle, swap wETH against agEUR, and then use these agEUR to redeem wBTC.
 
-## What happens if I ask to get a collateral that is not present in sufficient quantity against my stablecoins?
+## What happens if I want a collateral that is not present in sufficient quantity against my agEUR?
 
 Angle leaves the choice to its users as for the collateral type they can get when burning their stablecoins.
 
-It is however possible that there is an imbalance between the value of the different collateral pools backing a stablecoin. If 100 agEUR are backed by 10 DAI and 150 USDC, the transaction of a user asking DAI against 100 agEUR will fail because there are not enough DAI in reserves to reimburse the user.
+It is however possible that there is an imbalance between the value of the different collateral pools backing a stablecoin. If 100 agEUR are backed by 10 DAI and 150 USDC, the transactions of users asking DAI against 100 agEUR will fail because there are not enough DAI in reserves to reimburse the users.
+
+In this case, users will have to burn agEUR in exchange for USDC.
 
-## Do I still own the collateral I bring to the protocol to mint stablecoins?
+## Do I still own the collateral I bring to the protocol to mint agEUR?
 
-No. Unlike in Maker's system, Angle's protocol works with swaps from a user perspective, meaning that the collateral brought to get stablecoins does no longer belong to the person who brought it: it belongs to the protocol which keeps it in reserves and uses it to programmatically get some yield on it.
+No. Unlike in Maker's system, Angle's protocol works with swaps from a user perspective, meaning that the collateral brought to get stablecoins no longer belongs to the person who brought it: it belongs to the protocol which keeps it in reserves and uses it to back the minted stablecoin.
```

Comment:
does no longer belong

It's not correct to say no longer belongs

Reply:
Where did you find that? 
https://ludwig.guru/s/no+longer+belong+to